eessi/testsuite/eessi_mixin.py

```diff
@@ -202,6 +203,13 @@ def log_runtime_info(self):
             get_full_modpath = f'echo "FULL_MODULEPATH: $(module --location show {self.module_name})"'
             self.postrun_cmds.append(get_full_modpath)
 
+    @run_before('run', always_last=True)
+    def mixin_set_user_executable_opts(self):
+        "Override executable_opts with user_executable_opts if set on the cmd line"
+        if self.user_executable_opts:
+            self.executable_opts = [self.user_executable_opts]
```

Comment:
I think we should log also the old value. I.e. something like

```
log(f"Overwriting default executable_opts ('{self.executable_opts}') by executable_opts specified on command line ('{self.user_executable_opts}'))
```

Since we'd need the original value for `self.executable_opts`, this logging needs to be done _before_ the override.